packages/docusaurus-plugin-content-docs/src/__tests__/index.test.ts

```diff
@@ -584,6 +584,119 @@ describe('versioned website (community)', () => {
   });
 });
 
+describe('versioned website (community) with custom versions dir', () => {
```

Comment:
What about having a parametrized `loadSite({versionedDocsPath: "community-versions"})`?  

Wonder if it wouldn't simply our test a bit as you could reuse the `loadSite()` that already existed in the other describe?

I guess the initial idea was to have 1 describe per site fixture